src/H5ACmpio.c

```diff
@@ -2093,11 +2093,11 @@ H5AC__run_sync_point(H5F_t *f, int sync_point_op)
              (sync_point_op == H5AC_METADATA_WRITE_STRATEGY__DISTRIBUTED));
 
 #if H5AC_DEBUG_DIRTY_BYTES_CREATION
-    HDfprintf(stdout, "%d:H5AC_propagate...:%u: (u/uu/i/iu/r/ru) = %zu/%u/%zu/%u/%zu/%u\n", aux_ptr->mpi_rank,
+    HDfprintf(stdout, "%d:H5AC_propagate...:%u: (u/uu/i/iu/m/mu) = %zu/%u/%zu/%u/%zu/%u\n", aux_ptr->mpi_rank,
               aux_ptr->dirty_bytes_propagations, aux_ptr->unprotect_dirty_bytes,
               aux_ptr->unprotect_dirty_bytes_updates, aux_ptr->insert_dirty_bytes,
-              aux_ptr->insert_dirty_bytes_updates, aux_ptr->rename_dirty_bytes,
```

Comment:
The "rename_dirty_bytes" and "rename_dirty_bytes_updates" fields no longer exist in aux_ptr. Based on surrounding codes, looks like they have changed to "move_dirty_bytes" and "move_dirty_bytes_updates".